scripts/check-prepare.js

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env node
+
+process.exit(process.env.THEIA_SKIP_NPM_PREPARE ? 1 : 0);
```

Comment:
Without env variable it works as before, right? Could you add docs here why we need it?

Wonder also whether something like `yarn --skip-prepare` will be more convenient, as far as know yarn passes down all args to scripts.

Reply:
Will try with an argument, but I will name it `--theia-ignore-prepare` to explicitly show where it comes from (since there's a yarn builtin `--ignore-scripts`, don't want to confuse others).

Reply:
Tried, and arguments passed to `yarn --arg...` are not seen from the scripts.

I guess this is because the `prepare` script runs implicitly.

Went back with a environment variable, added a comment in the file.